code/controllers/subsystem/processing/ai_behaviors.dm

```diff
@@ -9,9 +9,9 @@ PROCESSING_SUBSYSTEM_DEF(ai_behaviors)
 	///List of all ai_behavior singletons, key is the typepath while assigned value is a newly created instance of the typepath. See SetupAIBehaviors()
 	var/list/ai_behaviors
 
-/datum/controller/subsystem/processing/ai_behaviors/Initialize(timeofday)
+/datum/controller/subsystem/processing/ai_behaviors/Initialize()
 	SetupAIBehaviors()
-	return ..()
+	return SS_INIT_SUCCESS
```

Comment:
does `/processing/` have an initialize?

Reply:
No sir, in fact it's SS_NO_INIT

Reply:
Same with timer, movement, and verb_manager. Fluids does init but the subtypes don't override.